crates/common/logged_command/src/logged_command.rs

```diff
@@ -28,6 +86,13 @@ impl LoggingChild {
     }
 }
 
+async fn send_sig_term(child: &Child) {
+    if let Some(pid) = child.id() {
```

Comment:
If `child.id()` returns `None`, this function does nothing. I think it should return error as it means no child process is there.

Reply:
fixed

Reply:
No the former code was correct. If [`child.id()`](https://docs.rs/tokio/latest/tokio/process/struct.Child.html#method.id) returns `None` this simply means that the child process no more running. This is not an error as our point was to kill it!

Reply:
Yes, thats right. I reverted it.